PocketKit/Tests/PocketKitTests/Recomendation/RecomendationExtensionTests.swift

```diff
@@ -0,0 +1,121 @@
+import XCTest
+import Sync
+
+@testable import Sync
+@testable import PocketKit
+
+class RecomendationExtensionTests: XCTestCase {
+    private var source: MockSource!
+    private var space: Space!
+
+    override func setUp() {
+        source = MockSource()
+        space = .testSpace()
+
+        continueAfterFailure = false
+    }
+
+    override func tearDownWithError() throws {
+        try space.clear()
+    }
+
+// MARK: - Builder Helpers
+
+    func buildItem(syndicatedArticle: SyndicatedArticle? = nil) -> Item {
+        return space.buildItem(
+            topImageURL: URL(string: "https://top-image-url.jpeg"),
+            excerpt: "Some item excerpt",
+            syndicatedArticle: syndicatedArticle
+        )
+    }
+
+    func subjectWithItem() -> Recommendation {
+        let recommendation = space.buildRecommendation(item: buildItem())
+        return recommendation
+    }
+
+    func subjectWithCuratedInfo() -> Recommendation {
+        let recommendation = space.buildRecommendation(
+            item: buildItem(),
+            imageURL: URL(string: "https://curated-info-image.jpeg"),
+            title: "A Curated Title",
+            excerpt: "Some curated excerpt"
+        )
+        return recommendation
+    }
```

Comment:
I feel like there's an opportunity for some cleanup, here. At bare minimum, `space.buildRecommendation(item: buildItem())` is reused. I'm wondering if we can consolidate this into a single API. Not required, just a potential for cleanup.

Reply:
Done.